package.json

```diff
@@ -14,17 +14,19 @@
     "@babel/core": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
+    "@emotion/core": "^10.0.14",
```

Comment:
This shouldn't be listed on `devDependencies`. 
The same for `react` and `react-dom`.

Reply:
Thank you